wagtail_wordpress_import/test/tests/test_block_builder.py

```diff
@@ -341,6 +341,16 @@ def test_get_abolute_src_slashes_at_start(self):
             "http://www.example.com/folder/fakeimage.jpg",
         )
 
+    def test_get_abolute_src_slashes_at_start_external_domain(self):
+        self.assertEqual(
+            get_absolute_src("//example.com/fakeimage.jpg", "https://www.example.com"),
+            "https://example.com/fakeimage.jpg",
+        )
+        self.assertEqual(
+            get_absolute_src("//example.com/fakeimage.jpg", "http://www.example.com"),
+            "http://example.com/fakeimage.jpg",
+        )
+
```

Comment:
**Suggestion**: Add a test case for an unrecognised extension on the local domain: `get_absolute_src("/fakeimage.badextension", "http://www.example.com")`